theme.toml

```diff
@@ -79,3 +84,13 @@ honey = "#efba5d"
 apricot = "#f47868"
 lightning = "#ffcd1c"
 delta = "#6F44F0"
+
+[glyphs]
+"diff.added" = '▌'
+"diff.removed.above" = '▘'
+"diff.removed.below" = '▖'
+"diff.modified" = '▐'
+diagnostic_hint = "●"
+diagnostic_info = "●"
+diagnostic_warning = "●"
+diagnostic_error = "●"
```

Comment:
I think the glyphs themselves should be configured via `config.toml` (maybe under something like `[vcs.signs]` and `[diagnostics.signs]` ?) so that themes only change the colors and not the symbols.

Reply:
While I don't mind either way, and I can understand the argument to keep a theme colour only, I can also see an argument that the glyphs are part of the theme.

Reply:
If we keep glyphs in themes and the user wants to change a symbol, they will have to copy over the theme they are using currently to their config folder and edit it, and they will have to do this for every new theme they try.

Reply:
Sure, but the exact same argument applies to a certain colour. And a theme maker would want their theme to look a certain way, glyphs included.

Reply:
I think it should be part of `theme.toml` because it is related more to theme rather than personal config.

Reply:
A good compromise that I can think of is to keep the glyphs in theme definitions and provide a way for _any_ theme item to be overridden via a config table like `[theme-override]`. Theme authors can specify colors and glyphs and users can selectively override any of it from their config without touching theme files.

Reply:
That sounds good to me.